src/components/Layout/Footer/index.tsx

```
@@ -19,6 +24,78 @@ const Footer: React.FC = () => {
+        toast.success(data.message || 'Successfully subscribed to our newsletter! 🎉');
+        setEmail(''); // Clear the input field
+        if (data.info) {
+          setTimeout(() => {
+            toast.success(data.info, { duration: 6000 });
+          }, 1000);
+        }
+      } else {
+        // Handle different error types
+        switch (response.status) {
+          case 429:
+            const retryAfter = data.retryAfter || 3600;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)